duckduckgo/iOS
# release enhanced homescreen

Task/Issue URL: https://app.asana.com/0/392891325557410/1111571693570069
Tech Design URL:
CC:

**Description**:

* Makes the enhanced home page available to users via a setting.
* Makes the cancel button version of the omni bar the default
* Cleans up variants
* Adds pixels to measure engagement 
* Moves refresh from menu to address bar
* Other minor UI updates as per design reviews

**Steps to test this PR**:

Clean Install:
1. Test each mode of the home page
1. After adding favorites return to a previous mode of the home page - the favorites should be migrated to bookmarks
1. The bookmarks view will show a favorites section when in favorites mode - the user can move bookmarks in  and between each section

Upgrade:
1. Install previous version with variant selected (may need to change code to make weightings in favour of favorites variant)
1. Upgrade to new version - should still show favorites
1. Repeat for centre search only

Ideally check on following device types:
1. iPad
1. iPhone
1. Small phone

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
